cli/torcx-generator.go

```diff
@@ -117,3 +129,29 @@ func lowerProfiles(commonCfg *torcx.CommonConfig) ([]string, error) {
 	}
 	return lowerProfiles, nil
 }
+
+// vendorProfileFromDockerFlag returns the vendor profile name, if any,
+// implied by the contents of /etc/coreos/docker-1.12.
+func vendorProfileFromDockerFlag(commonCfg *torcx.CommonConfig) string {
+	flagPath := filepath.Join(filepath.Dir(filepath.Clean(commonCfg.ConfDir)), "coreos", "docker-1.12")
+	flagBytes, err := ioutil.ReadFile(flagPath)
+	if err != nil {
+		if os.IsNotExist(err) {
+			logrus.Infof("no vendor profile selected by %s", flagPath)
+		} else {
+			logrus.Errorf("reading %s: %s", flagPath, err)
+		}
+		return ""
+	}
+
+	flag := strings.TrimSpace(string(flagBytes))
+	switch flag {
+	case "yes", "no":
+		profile := "docker-1.12-" + flag
```

Comment:
As I mentioned offline, I think it would be safer to actually check if this vendor-profile exists before trying to activate it. This is just a safety valve in case we miss synchronizing the removal of this logic with the removal of these profiles at the end of the transition. @bgilbert is positive that this won't happen and we have kola to ensure it, so this is a non-blocking concern at the moment.